mw-webapp/src/component/button/Button.module.scss

```diff
@@ -1,9 +1,19 @@
+@import "src/styles/variables.scss";
+
 .button {
-  width:  200px;
-  height: 50px;
+  width:  100px;
+  height: 40px;
   margin: 20px;
+  background: $color-btn;
```

Comment:
Bad naming. 
1. Don't cut words, example (color-btn -> button-color)
2. If it's background better use name button-background

Reply:
Done.